packages/web-components/fast-foundation/CEMToMarkdown.mjs

```diff
@@ -95,13 +95,30 @@ for(var i = 0, modulesLength = modules.length; i < modulesLength; i++)
         let markdown = customElementsManifestToMarkdown(componentManifest,
             {
                 headingOffset: 1,
-                private: 'hidden'
+                private: 'hidden',
+                omitDeclarations: ['exports'],
+                omitSections: ['static-methods']
             });
 
         // Replace our < and > markers with backticks and < >
         // This is necessary because customElementsManifestToMarkdown escapes the backticks during the conversion
         // and we don't want that because then docusaurus will see the tags as real tags instead of just text.
-        markdown = markdown.replaceAll("REPLACELT","`<").replaceAll("REPLACEGT",">`");
+        markdown = markdown.replaceAll("REPLACELT", "`<").replaceAll("REPLACEGT", ">`");
+
+        // Clean up some additional formatting issues
+        // Remove the file source header
+        markdown = markdown.replaceAll(/## `src.*`:/g, "");
+        // Fix escape of colon in urls
+        markdown = markdown.replaceAll("https\\:", "https:");
+        // Fix escape of . in some urls
+        markdown = markdown.replaceAll("www\\.w3", "www.w3");
+        // Fix escape of open bracket on links
+        markdown = markdown.replaceAll("\\[", "[");
+        // Fix escape of open paren on links
+        markdown = markdown.replaceAll("\\(", "(");
+
+        // Replace \| with 'or'
+        markdown = markdown.replaceAll("\\|","or");
```

Comment:
I think this is what's doing the replacement from my comment https://github.com/microsoft/fast/pull/5801/files#r843380813